src/hotspot/share/runtime/flags/jvmFlagConstraintsRuntime.cpp

```diff
@@ -127,3 +127,15 @@ JVMFlag::Error NUMAInterleaveGranularityConstraintFunc(size_t value, bool verbos
 
   return JVMFlag::SUCCESS;
 }
+
+JVMFlag::Error LargePageSizeInBytesConstraintFunc(size_t value, bool verbose) {
+  if (!is_power_of_2(value)) {
+    JVMFlag::printError(verbose,
+                        "LargePageSizeInBytes ( %zu ) must be "
+                        "a power of 2\n",
+                        value);
+    return JVMFlag::VIOLATES_CONSTRAINT;
+  } else {
+    return JVMFlag::SUCCESS;
+  }
+}
```

Comment:
I don't think you need `else` part here.

```suggestion
  } 
    
  return JVMFlag::SUCCESS;
}

```

Reply:
Makes sense. I was trying out stuff and no clue why I left it like this. Thanks for highlighting!